### DISCUSSION TITLE
Unable to add ubiquiti Dream Machine PRo

### DISCUSSION THREAD

**Discussion by KPtheitguy** [home-assistant/frontend#8398] [Configuration] [answered]
Hello, 
I was unable to add UDM pro onto home assistant using ubiquiti controller options, I created a local admin still same issue

**Comment by frenck** [comment, marked as answer]
Thank you for reaching out.  We use GitHub Discussions for tracking feature requests, not for providing support.

If you are in need of support, you should try our [Community Forum](https://community.home-assistant.io) or join our [Discord chat server](https://discord.gg/c5DvZ4e).

You are more likely to get the answer you are looking for in those places.

Thanks! 👍
